resources/dicts/thoughts/alive/kitten.json

```diff
@@ -26,28 +26,28 @@
     {
         "id": "gen_kit",
         "thoughts": [
-            "Plays moss ball by themselves",
-            "Wonders who their mentor will be",
+            "Plays moss ball by {PRONOUN/m_c/subject}",
+            "Wonders who {PRONOUN/m_c/subject} mentor will be",
             "Wants to take a nap",
             "Tries to sneak out of camp",
             "Is rolling around on the ground",
-            "Is chasing their tail",
+            "Is chasing {PRONOUN/m_c/subject} tail",
             "Is playing with a stick",
-            "Is nervous for their apprentice ceremony",
-            "Is excited for their apprentice ceremony",
+            "Is nervous for {PRONOUN/m_c/subject} apprentice ceremony",
+            "Is excited for {PRONOUN/m_c/subject} apprentice ceremony",
             "Is scared after having a nightmare",
             "Wants to snuggle",
             "Is bouncing around in excitement",
             "Whines about being hungry",
             "Is whining for milk",
-            "Is whimpering in their sleep",
+            "Is whimpering in {PRONOUN/m_c/subject} sleep",
             "Is trying to growl menacingly",
-            "Is adamantly refusing to take their nap",
+            "Is adamantly refusing to take {PRONOUN/m_c/subject} nap",
             "Is batting pebbles across the camp clearing",
             "Is crying after rough-housing too hard with the older cats",
-            "Is regretting eating the bug that they caught",
+            "Is regretting eating the bug that {PRONOUN/m_c/subject} caught",
             "Recently took a tumble off of a log",
-            "Is busy mastering a battle move they are performing incorrectly",
+            "Is busy mastering a battle move {PRONOUN/m_c/subject} {VERB/m_c/'ve/'s} performing incorrectly",
```

Comment:
I think you need 'been' between the verb and 'performing incorrectly', else it doesn't make much sense when read?